features/upgrade/samples_operator/upgrade.feature

```diff
@@ -0,0 +1,47 @@
+Feature: image-registry operator upgrade tests
+  # @author xiuwang@redhat.com
+  # @case_id OCP-22678
+  @upgrade-prepare
+  @users=upuser1,upuser2
+  @admin
+  Scenario: samples/openshift-controller-manager/imare-registry operators should be in correct status after upgrade
```

Comment:
1. Typo. imare -> image
2. Need ` - prepare` for upgrade prepare

Reply:
@liangxia @wzheng1 updated, please help review again, thx